README.md

```diff
@@ -1 +1,15 @@
 # Enceladus's Github Pages branch
+
+This branche's sole purpose is to serve as Documentation for the Enceladus project
+
+### Dependencies:
+- ruby >= 2.3.0
+
+### To build locally:
+```bash
+# In the root of the project
+$> git checkout gh-pages
+$> bundle install
```

Comment:
I guess this doesn't have to run always? Can you make split it to *run first time* and *run to build locally* or something like that.

Reply:
I would rather not. I have thought about it and checking that the moving dependencies (GitHub's pre-packaged gems) are always up to date is better then rethinking what went wrong. And the bundle install takes 2 secs if you are up to date with everything.